File: biolccc/src/examples/mass_calculation.cpp
#include <iostream>
#include <string>
#include <biolccc.h>

int main()
{
    std::string peptide("Ac-PEPTIDE-NH2");

    double averageMass = BioLCCC::calculateAverageMass(
        peptide, BioLCCC::rpAcnFaRod);
    double monoisotopicMass = BioLCCC::calculateMonoisotopicMass(
        peptide, BioLCCC::rpAcnFaRod);

    std::cout << "Average mass of " << peptide << " is " 
        << averageMass << " Da" << std::endl;
    std::cout << "Monoisotopic mass of " << peptide << " is " 
        << monoisotopicMass << " Da" << std::endl;

    return 0;
}

File: biolccc/src/examples/mass_calculation.py
import pyBioLCCC

peptide = 'Ac-PEPTIDE-NH2'

averageMass = pyBioLCCC.calculateAverageMass(
    peptide, pyBioLCCC.rpAcnFaRod)
monoisotopicMass = pyBioLCCC.calculateMonoisotopicMass(
    peptide, pyBioLCCC.rpAcnFaRod)

print 'The average mass of', peptide, 'is', averageMass, 'Da'
print 'The monoisotopic mass of', peptide, 'is', monoisotopicMass, 'Da'
